src/components/index.ts

```diff
@@ -21,3 +21,4 @@ export { InlineSwitch, CertificationKey } from '@grafana/ui';
 export * from './QueryEditor';
 export * from './ConfigEditor';
 export { CustomHeadersSettings } from './CustomHeadersSettings/CustomHeadersSettings';
+export { ApiQueryEditor } from './ApiQueryEditor/ApiQueryEditor';
```

Comment:
We should probably also export the types so that we can consume them in plugins.